modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceCheckout.testcase

```diff
@@ -251,6 +251,95 @@ definition {
 		}
 	}
 
+	test AddEmptyOrderFromAUserAndCheckoutFromAnotherUserViaMiniCart {
+		property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}company.security.strangers.verify=false";
```

Comment:
Please remove this line as the same line is already set at the definition level. Thanks